rspace/src/main/scala/coop/rchain/rspace/IStore.scala

```diff
@@ -104,6 +106,7 @@ trait IStore[C, P, A, K] {
 
   protected def processTrieUpdate(update: TrieUpdate[C, P, A, K]): Unit
 
+  @SuppressWarnings(Array("org.wartremover.warts.Var"))
   private[rspace] def getAndClearTrieUpdates(): Seq[TrieUpdate[C, P, A, K]] =
```

Comment:
I don't see a `var` in this or the two previous methods. I think those suppressions are unneeded :)

Reply:
This one is false-positive and unfortunate this is why supress is here